src/applications/vaos/services/healthcare-service/transformers.js

```diff
@@ -15,121 +15,14 @@
  */
 export function transformAvailableClinic(facilityId, typeOfCareId, clinic) {
   return {
+    // ID: VA facility site code and clinic id
     id: `${clinic.siteCode}_${clinic.clinicId}`,
```

Comment:
I left this as is but I'm wondering if it should be broken up or just mapped to `clinic.clinicId`. I believe `clinic.siteCode` is the same as `facilityId`?

Reply:
They're sometimes the same, but not always. `facilityId` should be the full 5 digit id of the VA location, while siteCode is the 3 digit VistA site id.

A clinic is uniquely identified by a combination of site code and clinic id, so I'm hesitant to set it to just clinic id, since we could have clinic id collisions across sites. That probably wouldn't cause any trouble in the form flow, but could make things harder on the list pages if we're looking at clinic data.